pkg/api/validation/dynakube/image.go

```diff
@@ -47,18 +42,12 @@ func imageFieldHasTenantImage(_ context.Context, _ *Validator, dk *dynakube.Dyna
 	return strings.Join(messages, ";")
 }
 
-func checkImageField(image, section, disallowedHost string) (errorMsg string) {
+func checkImageField(image, section string) (errorMsg string) {
 	if image != "" {
-		ref, err := name.ParseReference(image)
+		_, err := name.ParseReference(image)
 		if err != nil {
 			return fmt.Sprintf(errorUnparsableImageRef, section)
 		}
-
-		refUrl, _ := url.Parse(ref.Name())
-
-		if refUrl.Host == disallowedHost {
```

Comment:
just to make sure we all aware that we open gate for some security problems in future :) 
cc @aorcholski

Reply:
what security problems?

Reply:
i mean you can use any registry which you want and we will pull from it and try to run it :) 

it's only potential problem :) nothing against implementation.

Reply:
> i mean you can use any registry which you want and we will pull from it and try to run it :)

you could already do this, the only registry you couldn't use is the official Dynatrace Environment's registry ([look at how it was used](https://github.com/Dynatrace/dynatrace-operator/pull/4220/files#diff-a003644eb8d11e98c18628b0ffbdbd45e53e3a35ed4783c9985e52b1a794babbL41))

Reply:
oh i misread code, my bad.